dartdoc_test/lib/src/extractor.dart

```diff
@@ -50,14 +52,19 @@ class DocumentationCodeSample {
     this.imports = const [],
   });
 
-  String get wrappedCode {
-    final fileName = comment.span.sourceUrl;
+  String wrappedCode(Directory testDir) {
+    var fileName = comment.span.sourceUrl;
```

Comment:
We can probably use `final` here.

Reply:
Thank you for reviewing. I applied your suggetions.